scripts/src/test/bash/all-tests

```diff
@@ -12,15 +14,17 @@ do
   mkdir -p "${testcase}/conf"
   cd "${testcase}"
   echo "export M2_REPO=~/.m2/repository" > "conf/devon.properties"
-  "${testpath}"
+  "${testpath}" &> errorLog
```

Comment:
so on success we still get the log. Could cause some spam and make it harder to read but with the coloring should be fine now.